xunit-autolabeler-v2/ast_parser/python/source_parser.py

```diff
@@ -151,3 +154,11 @@ def get_top_level_methods(source_path: str) -> List[Any]:
             f'\t{str(err)}\n')
 
         return []
+    except SyntaxError as err:
+        # Fail gracefully if a file doesn't use py3-compliant syntax.
+        sys.stderr.write(
+            f'WARNING: could not parse file: {source_path}\n')
+        sys.stderr.write(
+            f'\t{str(err)}\n')
+
+        return []
```

Comment:
I'm a bit confused about the changes to this private method (_get_ending_line), I only see 2 return branches and both return an empty list? And based on the docstring it's supposed to return an int (line number expression ends)?

Reply:
Sorry - I think the second diff block is part of a different method (`get_top_level_methods`)?

Reply:
🤦‍♂️ my bad, the UI was hiding a bunch of lines and I didn't realized. Sorry.